src/showplanner/libraries.tsx

```diff
@@ -21,6 +36,131 @@ type searchingStateEnum =
   | "results"
   | "no-results";
 
+export function LibraryColumn() {
+  const [sauce, setSauce] = useState("None");
+  const dispatch = useDispatch();
+  const { auxPlaylists, managedPlaylists, userPlaylists } = useSelector(
+    (state: RootState) => state.showplan
```

Comment:
Consider more granular selector scope?

Reply:
This was already here, from the re-organising. Will try.